core/app/[locale]/(default)/cart/page.tsx

```diff
@@ -88,6 +93,14 @@ export default async function Cart({ params }: Props) {
     );
   }
 
+  const walletButtons = await getPaymentWallets({
+    filters: {
+      cartEntityId: cartId,
+    },
+  });
+
+  const walletButtonsInitOptions = await createWalletButtonsInitOptions(walletButtons, cart);
```

Comment:
Instead of these being waterfall requests, can you see about using our `Streamable` pattern here?